roles/elasticsearch/tasks/main.yml

```diff
@@ -84,7 +84,7 @@
 - name: Ensure PCP Elasticsearch export is running and enabled on boot
   service:
     name: pcp2elasticsearch
-    state: started
+    state: "{{ 'started' if ansible_connection != 'buildah' else omit }}"
```

Comment:
This warrants some discussion: Checking `ansible_connection` here works for the "classic" qemu/real iron and even "system podman container" case, but it errs in the direction of "assuming booted system". I.e. if you run it against a podman connection on a non-system container, it will also fail.

In system-roles, we use a more precise `systemctl is-system-running` call instead. However, this would either have to be copied to all the different roles here (which is expensive and also redundant), or passed in from outside, with `__metrics_is_booted | d(true)`. The latter would be fine for me as well, it just makes the role a bit "leaky".

Reply:
I.e. you'd prefer that exact but expensive change over `ansible_connection != 'buildah'` (that *is* independent, just not completely precise). Will do tomorrow then (EOD, sorry)

Reply:
Ack, I've done that.